app/models/site_settings.rb

```diff
@@ -13,11 +13,29 @@
 #
 
 class SiteSettings < ApplicationRecord
+  store_accessor :prices, :electricity_price, :solar_export_price, :gas_price, :oil_price
+  validates :electricity_price, :solar_export_price, :gas_price, :oil_price, numericality: { only_float: true, allow_blank: true }
```

Comment:
We're allowing blank values here. Shouldn't we make sure there's always a price set?

Reply:
:+1:

Reply:
So I think this is still outstanding. There must always be a non-zero price?